server/services/WeekPhsaseService/weekPhasesService.js

```diff
@@ -0,0 +1,39 @@
+"use strict"
+
+var cron = require('node-cron');
```

Comment:
Importing this module multiple time will result in multiple instances of it running. Clone the dao behavior.